server/events/project_command_runner.go

```diff
@@ -238,7 +238,7 @@ func (p *DefaultProjectCommandRunner) doApply(ctx models.ProjectCommandContext)
 			if err != nil {
 				return "", "", errors.Wrap(err, "checking if pull request is mergeable")
 			}
-			if !mergeable {
+			if !ctx.Mergeable && !mergeable {
```

Comment:
Why do we bother making the call again when we've already made it at the beginning?